apps/api/src/modules/resume/resume.respository.ts

```diff
@@ -0,0 +1,29 @@
+import { JwtDto, PostResumeDto } from '@modela/dtos'
+import { Injectable, NotFoundException } from '@nestjs/common'
+import { PrismaService } from 'src/database/prisma.service'
+
+@Injectable()
+export class ResumeRepository {
+  constructor(private prisma: PrismaService) {}
+
+  async createResume(postResumeDto: PostResumeDto, user: JwtDto) {
+    const actor = await this.prisma.actor.findUnique({
+      where: { actorId: user.userId },
+    })
+    if (!actor) {
+      throw new NotFoundException('Actor not found')
+    }
```

Comment:
@miello do we need to do this

Reply:
Can we do checking on ResumeService instead ?